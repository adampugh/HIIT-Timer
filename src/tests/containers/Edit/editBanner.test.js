import React from "react";
import { shallow } from "enzyme"; 

import EditBanner from "../../../containers/Edit/editBanner";

test("should render editBanner correctly", () => {
    const wrapper = shallow(<EditBanner />);
    expect(wrapper).toMatchSnapshot();
});