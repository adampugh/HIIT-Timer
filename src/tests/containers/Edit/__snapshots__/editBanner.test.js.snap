// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render editBanner correctly 1`] = `
<div
  className="bannerText"
>
  <div
    className="bannerText__container"
  >
    <h2>
      Add exercises to create a full HIIT workout. Click the + button below to add a new exercise.
    </h2>
    <img
      alt="arm icon"
      className="bannerText__image"
      src="dumbbell-icon.png"
    />
  </div>
</div>
`;
